File: Module.Frontend.TPM/Client/model/tpm/budgetitem/HistoricalBudgetItem.js
Ext.define('App.model.tpm.budgetitem.HistoricalBudgetItem', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'BudgetItem',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true },
        { name: '_Operation', type: 'string', isDefault: true },        
        { name: 'BudgetId', hidden: true, isDefault: true },
        { name: 'BudgetName', type: 'string' },
        { name: 'Name', type: 'string', hidden: false, isDefault: true },
        { name: 'ButtonColor', useNull: false, type: 'string', hidden: false, isDefault: true },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalBudgetItems',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalBudgetItem.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;
namespace Module.Persist.TPM.Model.History
{
  [AssociatedWith(typeof(BudgetItem ))]
  public class HistoricalBudgetItem : BaseHistoricalEntity<System.Guid>
{
        public Guid BudgetId { get; set; }
        public string Name {get; set; }
        public string BudgetName { get; set; }
        public string Description_ru { get; set; }
        public string ButtonColor { get; set; }
    }
}
